Changelog.md

```diff
@@ -0,0 +1,4 @@
+
+## 0.11.5
```

Comment:
Don't bump the version or write the changelog entry please. My practice is to do that after merge, in case multiple changes/PRs are going into one new version..

Reply:
Okay, sounds good.  I'll take that out.

Reply:
Okay, I've removed the commit where I add a changelog and bump the version number.